src/js/components/ServiceList.js

```diff
@@ -40,7 +40,7 @@ let ServiceList = React.createClass({
   },
 
   getServices: function (services, healthProcessed) {
-    return _.map(services, function (service) {
+    return services.map(function (service) {
```

Comment:
`map` doesn't take a 2nd argument. this whole block of code should not be working right now (i haven't tested)

use `=>`